imageprocessor/processorcommand/gm.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/Imgur/mandible/imageprocessor/thumbType"
 )
 
-const GM_COMMAND = "convert"
+const GM_COMMAND = "gm convert"
```

Comment:
Hrm - yeah - I think i'll just add "convert" as an arg to each of those commands. That way `runProcessorCommand` stays generic and we can use other parts of `gm` easily if we want.